File: Java/MathematicalPrograms/3DShapes/AreaAndPerimeterOfCylinder.java
import java.util.Scanner;
import java.lang.Math;

public class AreaAndPerimeterOfCylinder {
	public static void main(String[] args) {
		Scanner myObj = new Scanner(System.in);
		System.out.println("Enter the Radius");
		int radius = myObj.nextInt();
		System.out.println("Enter the Height");
		int height = myObj.nextInt();
		double c = 2*Math.PI*radius;
		System.out.println("Perimeter of the Bottom Circle is " + c + " cm");
		double a = Math.PI*Math.pow(radius, 2);
		System.out.println("Area of the Bottom Circle is " + a + " sq.cm.");
		double tsa = 2*Math.PI*radius*(radius+height);
		double lsa = 2*Math.PI*radius*height;
		System.out.println("Total Surface Area of the Cylinder is " + tsa + " sq.cm.");
		System.out.println("Lateral Surface Area of the Cylinder is " + lsa + " sq.cm.");
	}
}

File: Python/MathematicalPrograms/3DShapes/AreaAndPerimeterOfCylinder.py
import math
pi=math.pi
r=int(input('Enter the Radius'))
h=int(input('Enter the Height'))
c=2*pi*r
print ('Perimeter of the Bottom Circle is', c, "cm")
a=pi*r**2
print ('Area of the Bottom Circle is', a, "sq.cm.")
tsa=2*pi*r*(r+h)
print ('Total Surface Area of the Cylinder is', tsa, "sq.cm.")
lsa=2*pi*r*h
print ('Lateral Surface Area of the Cylinder is', lsa, "sq.cm.")